project.py

```diff
@@ -0,0 +1,50 @@
+import pandas as pd
+import csv
+import numpy as np
+
+df = pd.read_csv('real-estate.csv')
+
+print(df.to_string())
+print(df.dtypes)
+
+rank = 0
+file = open('real-estate.csv','r')
+
+csv_reader = csv.reader(file,delimiter=',')
+
+csv_list = list(csv_reader)
+
+file1 = open('sorted_Real_Estate.csv','w+')
+
+rows = csv_list[0]
+file1.write("Rank,")
+for i in rows:
```

Comment:
Move this into its own function

Reply:
I have worked on the changes and I have also implemented the search algorithm